V1Documentation.md

```diff
@@ -115,51 +113,17 @@ Raygun.detach();
 
 **IE8**
 
-If you are serving your site over HTTP and want IE8 to be able to submit JavaScript errors then you will
-need to set the following setting which will allow IE8 to submit the error over HTTP. Otherwise the provider
-will only submit over HTTPS which IE8 will not allow while being served over HTTP.
+If you are serving your site over HTTP and want IE8 to be able to submit JavaScript errors then you will need to set the following setting which will allow IE8 to submit the error over HTTP. Otherwise the provider will only submit over HTTPS which IE8 will not allow while being served over HTTP.
 
 ```javascript
-// V2
-rg4js('options', {
-  allowInsecureSubmissions: true
-});
-
-// V1
 Raygun.init('yourApiKey', { allowInsecureSubmissions: true });
 ```
 
-## Documentation
-
-### Differences between Raygun4JS V1 and V2
```

Comment:
This block should also be present, but reworded to highlight the difference (shortly) for V1